ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -158,7 +160,9 @@ def select_vlan_ports(self):
     def down_vlan_ports(self):
         # extract the selected vlan ports and mark them down
         for item in self.down_vlan_info:
-            self.vlan_ports.remove(item[1])
+            self.vlan_ports = [port for port in self.vlan_ports if port != item[1]]
```

Comment:
Is this needed? Can it not just be `self.vlan_ports.remove(item[1])`?

Reply:
In the cases that there are ports overlapped between VLANs, `self.vlan_ports` may contain multiples occurrences of the same port. This change aims to remove all in case of overlapped VLANs, since `remove()` function only removes the first occurrence in the list.